pkg/api/cluster.go

```diff
@@ -884,6 +884,24 @@ func (c *Cluster) AvailabilityZones() []string {
 	return result
 }
 
+func (c *Cluster) ControllerFeatureGates() FeatureGates {
```

Comment:
This is great work, thanks, but I wonder if it wouldn't be better to follow kubernetes here and also set these on by default too?  We could offer functionality to allow a user to choose to disable them.   Would you mind updating to do this?

Reply:
Done